src/pages/organizers/[organizerId]/ownerships/index.page.tsx

```diff
@@ -90,6 +97,25 @@ const Ownership = () => {
     },
   });
 
+  const approveRequestedOwnership = async (ownershipId: string) => {
+    setIsSuccessAlertVisible(true);
+    setIsOpen(false);
+
+    return approveOwnershipRequestMutation.mutateAsync({ ownershipId });
+  };
+
+  const requestOwnership = useRequestOwnershipMutation({
+    onError: async (error: FetchError<ErrorBody>) => {
+      if (error.body.status === 409) {
+        const ownershipId = error.body.detail.split('with id ')[1];
+        await approveRequestedOwnership(ownershipId);
+      } else {
+        setError('email', { message: error.body.detail || error.title });
```

Comment:
@simon-debruijn I did this originally to cascade the email errors back to the user UI, but ended up reverting it to a generic message but it allowed users to probe for existing emails in the system, but now it's sometimes cryptic why an email is rejected. Do you think we should show the actual errors?